My Gatsby site couldn't compile due to this package.

This is the error message I get when trying to compile my Gatsby site:

```
Module not found: Error: Invalid "exports" target "index.js" defined for "." in the package config [...]\node_modules\aframe-extras\package.json, targets must start with "./"
```
*(I just truncated the directory of my project with [...].)*

Upon checking, the `package.json` of this package, on line 9, reads `".": "index.js",`, which causes the error.

This package is a dependency of `react-force-graph-3d`, which I use for my Gatsby site.

I believe this is a simple fix of changing "." to "./".

--- a/package.json
+++ b/package.json
@@ -4,9 +4,9 @@
   "description": "Add-ons and examples for A-Frame VR.",
   "author": "Don McCurdy <dm@donmccurdy.com>",
   "license": "MIT",
-  "main": "index.js",
+  "main": "./index.js",
   "exports": {
-    ".": "index.js",
+    ".": "./index.js",
     "./controls/*": "./src/controls/*",
     "./loaders/*": "./src/loaders/*",
     "./misc/*": "./src/misc/*",

